quesma/util/utils.go

```diff
@@ -852,6 +852,9 @@ func replaceNonAlphabetic(str string) string {
 }
 
 func FieldToColumnEncoder(field string) string {
+	if len(field) == 0 {
```

Comment:
This function definitely needs a comment like
```
FieldToColumnEncoder takes publicly-facing Elasticsearch field name and using well-defined algorithm converts it to "legit" column name in ClickHouse. This encoding needs to happen because we don't want to have dots, colons or any other non-alphanumerical characters in columns names.  
```